config/config_test.go

```diff
@@ -0,0 +1,168 @@
+package config
```

Comment:
Just a question on process. Do you use TDD to generate these test files, or do you write the tests immediately after the file under test?